nrt/nrt.py

```diff
@@ -531,6 +534,24 @@ def main():
 
         log_message += " pattern filter"
 
+    directory_failed = []
+    if (options.failed == True):
+        for root, dirs, files in os.walk(output_dir):
+            for file in files:
+                if "info.txt" == file:
+                    test_case = nrtDiff.TestCase("dummy")
+                    test_case.get_case_info(os.path.join(root, file))
+                    if test_case.status_ == "KO":
```

Comment:
I would propose to also add `or test_case.cmp_status == "KO" `as reference comparison failure is also a common reason to rerun nrt